### ISSUE TITLE
Copy button doesn't seem to work on Mac/Chrome

### ISSUE THREAD

**Issue by andupotorac** [CodeZombie/latentcoupleregionmapper#1]
Chrome Version 114.0.5735.133 (Official Build) (arm64)
MacOS 13.5 Beta (22G5048d)

Nothing happens on click on either button. No errors in the console either.

**Comment by CodeZombie** [comment]
Good catch - This is because the copy routine uses the `Clipboard.writeText` api which will silently fail if the page isn't be served via https.
This is obviously no good, I'll do some more digging later today to see if there's a more compatible alternative and make the fix.

Thanks for finding this!

**Comment by andupotorac** [comment]
Awesome!

**Comment by andupotorac** [comment]
@CodeZombie BTW you know what would be useful? A way to export, import a predefined composition, because these don't seem to get saved in the PNG info on SD, so then we lose them after we create the images.

**Comment by andupotorac** [comment]
> 

Please disregard this. It turns out there's a way to save it in PNG info already. Tried and it works:
  1. https://github.com/opparco/stable-diffusion-webui-two-shot/issues/43
